addons/modules/functions/fnc_modulePatrolArea.sqf

```diff
@@ -51,7 +52,7 @@ if !(alive _unit) exitWith {
         private _speedMode = ["LIMITED", "NORMAL"] select (_behaviour > 1);
```

Comment:
Should also be changed, not sure what speed mode makes most sense for careless behaviour.

Reply:
I would assume limited for the most part.

Reply:
Careless behavior is accompanied by Limited speed. Though maybe also having Relaxed be accompanied by Limited speed be a good idea

Reply:
Yes, change it to `_behaviour > 2`.